optuna_dashboard/_app.py

```diff
@@ -449,6 +452,48 @@ def save_trial_note(study_id: int, trial_id: int) -> dict[str, Any]:
         response.status = 204  # No content
         return {}
 
+    @app.get("/csv/<study_id:int>")
+    def download_csv(study_id: int) -> BottleViewReturn:
+        # Create a CSV file
+        summary = get_study_summary(storage, study_id)
+        if summary is None:
+            response.status = 404  # Not found
+            return {"reason": f"study_id={study_id} is not found"}
+        trials = get_trials(storage, study_id)
+        if len(trials) == 0:
+            return {"reason": f"study_id={study_id} has no trials"}
```

Comment:
As mentioned, after all the modification I commented, this line is not necessary anymore.
```suggestion
```